From 73c0992a36bc13e9bb373f98ba246dfc1e29a393 Mon Sep 17 00:00:00 2001
From: Robert Marko <robimarko@gmail.com>
Date: Tue, 7 Nov 2023 14:21:43 +0100
Subject: [PATCH 3/3] Revert "[qca-ssdk]: enable invoking fdb del function for
 kernel6.1"

This reverts commit a86765ee6a87145f64344f4872cfe8c5e629dd62.

Upstream kernel does not have this call present at all, we currently
also dont use it, so revert the commit enabling it.

Signed-off-by: Robert Marko <robimarko@gmail.com>
---
 src/ref/ref_acl.c | 4 ++++
 1 file changed, 4 insertions(+)

--- a/src/ref/ref_acl.c
+++ b/src/ref/ref_acl.c
@@ -229,7 +229,11 @@ _ref_acl_mac_entry_create_rule(a_uint32_
 		eth_dev = dev_get_by_name(&init_net, entry->ifname);
 		if (eth_dev)
 		{
+#if (LINUX_VERSION_CODE >= KERNEL_VERSION(6, 1, 0))
+			// TODO: replace with corresponding ver
+#else
 			br_fdb_delete_by_netdev(eth_dev, entry->src_mac.uc, 0);
+#endif
 			dev_put(eth_dev);
 		}
 	}
